README.md

```diff
@@ -1,2 +1,168 @@
+
+### LocalizedText
+
+A LocalizedText object shall be used whenever text requires localization or may require in the future. The object may either simply be a single string (in the case of localizations not being available yet) or a dictionary with string keys and string values. If LocalizedText is represented by a string-string dictionary, the keys represent language-codes following the ISO 639-1 (or if necessary ISO 639-1) standard and the respective text in the given language as value.
+
+A LocalizedText object cannot be used in FHIR-conforming types due to its incompatibility with the standard. FHIR types commonly contain text in one language only to be specified using the `language` property.
+
+## /medications
+
+In this section, we describe information regarding all the medications to be specified in the Engage-HF context. These medications may be used by a clinician for medication requests to a patient (/users/$userId$/medicationRequests) or contra-indications (/users/$userId$/allergyIntolerances).
+
+### /medications/$medicationId$
+
+Based on [FHIR Medication](https://hl7.org/fhir/R4B/medication.html):
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|meta|Meta|-|[Resource](https://hl7.org/fhir/R4B/resource.html#Meta): Metadata about the resource|
+|identifier|list of Identifier|-|Business identifier for this medication|
+|extension|list of Extension|-|See /medications/$medicationId$/extension for possible values|
+
+Technically, we could use [FHIR MedicationKnowledge](https://hl7.org/fhir/R4B/medicationknowledge.html), but it doesn't seem to be ready for use - or at least take over the `medicineClassification` property from it.
+
+### /medications/$medicationId$/extension
+
+Based on the [Extension](https://hl7.org/fhir/R4B/extensibility.html#Extension) format specified in FHIR, a medication may contain a list of these following properties. Each property will need to get a url assigned to fit the FHIR data format.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|medicationClass|string|e.g. "BETABLOCKER"|A medicationClassId referring to a medicationClass specified in /medicationClasses/$medicationClassId$. One medication object may contain multiple medicationClass extension properties.|
+|minimumDailyDose|double|e.g. 6.25|Unit: mg/day. May only occur once.|
+|targetDailyDose|double|e.g. 50.0|Unit: mg/day. May only occur once.|
+
+Q: Do we possibly need a connection from a single medication to a video?
+
+### /medicationClasses/$medicationClassId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|e.g. "BETABLOCKER"||
+|name|LocalizedText|-|A name for a given medicationClass to be displayed to a user.|
+|videoSectionId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+|videoId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
```

Comment:
Could we combine this in a single "URL" or relative path that we used to identify videos?

Reply:
✅  Changed it to be a single `videoPath` property.